src/rollup/index.ts

```diff
@@ -362,11 +301,12 @@ export default function rollup(
 										}
 									})
 									// second render chunks given known names
-									.then(() => {
-										return Promise.all(
+									.then(() =>
+										Promise.all(
 											chunks.map(chunk => {
 												return chunk.render(outputOptions, addons).then(rendered => {
 													const output = {
+														name: chunk.id,
```

Comment:
We could possibly call this `filename` as well. The idea is that it is possible to determine the output file path from this using `path.resolve(outputOptions.dir || process.cwd())` to reliably output eg a CSS file per chunk,

Reply:
Another reason we might still want a name when not writing is because we do have file names for chunking, but its only the single file build that doesn't have a rule here. So if we want the APIs to be unified at some point, ideally we can always assign a name - so that was the thinking behind using the `input` fallback.

Will change to file for now and keep it always-populated, but we can reconsider this.

Reply:
The other thing is `name` aligns with the naming of `chunkNames` and `entryNames` which are actually naming files in the same space... so would be good to try converge on terminology here actually.